### DISCUSSION TITLE
Webcam?

### DISCUSSION THREAD

**Discussion by GrimeyPlopFlop** [Monika-After-Story/MonikaModDev#9021] [QA - tech support + general] [answered]
Is there any webcam feature for MAS? One of Monika's topics talks about how she wish there was a webcam she could access to see me, but I'm fairly new to this community so I don't know if that's a thing in the game. Thank you!

**Comment by Livingfigure** [comment, marked as answer]
I guess the answer is yes and no.There do have a submod which can allow monika to use webcam.But, unfortunately, that submod don't always work and the creators of MAS think face detection is too complex. So they won't make such a option for now. Pardon for bad english as i'm not a native speaker.
